nordic-epub3-dtbook-migrator/src/test/xprocspec/html-to-epub3.step.xprocspec

```diff
@@ -71,10 +75,10 @@
                 <d:file href="EPUB/nav.ncx" method="xml" indent="true" omit-xml-declaration="false" media-type="application/x-dtbncx+xml"/>
             </x:document>
             <x:document type="inline" base-uri="temp-dir">
-                <d:file href="EPUB/package.opf" method="xml" indent="true" omit-xml-declaration="false" media-type="application/oebps-package+xml"/>
+                <d:file href="EPUB/nav.xhtml" method="xhtml" indent="true" omit-xml-declaration="false" media-type="application/xhtml+xml" doctype="&lt;!DOCTYPE html&gt;"/>
```

Comment:
Any reason why the navigation document had to be first? (just curious)

Reply:
I can't immediately pinpoint exactly what caused this change, but it probably happened somewhere in the refactoring of commit 1c0426484223c14398becf7cd0dbe24ccd6ca5c9. Do you really want to know? The fileset XML before/after is equivalent, so...

Reply:
Nah it doesn't really matter.